docs/resources/custom_framework.md

```diff
@@ -0,0 +1,59 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
```

Comment:
This was auto generated and the cassettes were also auto generated. Everything else was written

Reply:
the fix for this: manually editing the file since the api-spec channel ack'ed this and said a solution still needs to go out